skills/csharp/calendarskill/Dialogs/CreateEventDialog.cs

```diff
@@ -617,17 +617,18 @@ private async Task<DialogTurnResult> CreateEventAsync(WaterfallStepContext sc, C
                     Title = state.MeetingInfo.Title,
                     Content = state.MeetingInfo.Content,
                     Attendees = state.MeetingInfo.ContactInfor.Contacts,
-                    StartTime = (DateTime)state.MeetingInfo.StartDateTime,
-                    EndTime = (DateTime)state.MeetingInfo.EndDateTime,
-                    TimeZone = TimeZoneInfo.Utc,
+                    StartTime = TimeConverter.ConvertUtcToUserTime((DateTime)state.MeetingInfo.StartDateTime, state.GetUserTimeZone()),
```

Comment:
>TimeConverter.ConvertUtcToUserTime((DateTime)state.MeetingInfo.StartDateTime, state.GetUserTimeZone()), [](start = 32, length = 103)

This feels strange to me. Shouldn't the point of storing the Time Zone separately mean that we should just get a DateTime that we can take to be in the same time zone as specified in the state?

Reply:
Yes. But according to the previous design, the StartDateTime and EndDateTime should be UTC time and be used in many places. If we change it to specified timezone, we need to change a lot of code. I am not sure should we take so many changes on this bug. So I choose a simpler way to fix it.